bin/functional-tests/test_config.py

```diff
@@ -152,12 +152,11 @@ def test_prometheus_config_reloader_works(prometheus, kube_client):
         print(f"Exception when calling CoreV1Api->patch_namespaced_config_map: {e}\n")
 
     # This can take more than a minute.
-    for i in range(12):
+    for _ in range(12):
         data = prometheus.check_output(
             "wget -qO- http://localhost:9090/api/v1/status/config"
         )
         j_parsed = json.loads(data)
-        # print(parsed['data']['yaml']['config']['global'])
         y_parsed = yaml.safe_load(j_parsed["data"]["yaml"])
         if y_parsed["global"]["scrape_interval"] != "30s":
             print(y_parsed["global"]["scrape_interval"])
```

Comment:
in general, i think it would be better to use a logging library instead of print statements. but while we're using prints, should we print on line 165 as well? just to show that we're retrying?

Reply:
what would the benefit of a logging library be in a test case?

Reply:
tbh it probably depends on the environment. at my prev company, i had logging https://pypi.org/project/pytest-html/. html report was really nice for our UI tests since we were able to add screenshots next to our logs, but we probably don't need all this for this env